README.md

```diff
@@ -119,10 +119,9 @@ on how to write meaningful descriptions for your contributions.
 To do the upmerge run the following commands
 
 ```
-git checkout 4.0 && git fetch && git reset --hard origin/4.0 && git merge --no-ff --no-commit origin/2.x --strategy-option=ours
-git checkout 5.0 && git fetch && git reset --hard origin/5.0 && git merge --no-ff --no-commit origin/4.0 --strategy-option=ours
+git checkout 5.1 && git fetch && git reset --hard origin/5.1 && git merge --no-ff --no-commit origin/5.0 --strategy-option=ours
```

Comment:
Hey sorry for the late review, but what about upmerging from 4.x to 5.0?

Reply:
Did not add that because it was not there before and looks logic to me that you just need to change the versions

Reply:
Why, there was this line: `git checkout 5.0 && git fetch && git reset --hard origin/5.0 && git merge --no-ff --no-commit origin/4.0 --strategy-option=ours`